.github/workflows/npm-publish.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: microsoft/playwright-github-action@v1
       - uses: actions/setup-node@v1
```

Comment:
I think this is good, but I noticed the order of `uses` in the CI script is

```
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
      - uses: microsoft/playwright-github-action@v1
```

Should we make them the same order for consistency?

Reply:
So I think that was how it was (effectively) before I made the last change.  On the topic of consistency, maybe what I'll do is model it closer to how we do the CI workflow though.

Reply:
Cool done 😎